packages/components/src/styles/components/code-editor/index.scss

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CODE-EDITOR
+//
+
+@use "theme";
+
+.hds-code-editor {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: var(--hds-code-editor-color-surface-primary);
+  border: 1px solid var(--hds-code-editor-color-border-strong);
+  border-radius: 6px;
+
+  &.hds-code-editor--is-full-screen {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
Tip: You can replace the top, right, etc. styles with `inset: 0;` to set them all at once.

https://developer.mozilla.org/en-US/docs/Web/CSS/inset